components/spluseins-impressum.vue

```diff
@@ -0,0 +1,120 @@
+<template>
+  <span class="text-xs-center">
+    <v-dialog
+      v-model="active"
+      width="800"
+    >
+      <span 
+        slot="activator"
+        class="impressum"
+        white--text>
+        Impressum
+      </span>
+      <v-card>
+        <v-card-title
+          class="headline black white--text"
+          primary-title
+        >
+          Impressum
+        </v-card-title>
+
+        <v-card-text>
+          <p>Angaben gemäß § 5 TMG</p>
+
+          <p>SplusEins Team</p>
+
+          <p>Am Exer 2</p>
+
+          <p>38302 Wolfenbüttel</p>
+
+          <p><b>Vertreten durch:</b></p>
+
+          <p>Finn Kaiser</p>
+
+          <p>Jan Fasel</p>
+
+          <p>Fabain Pfütsch</p>
+
+          <p>Timo Haumann</p>
+
+          <p><b>Kontakt:</b></p>
+
+          <p>E-Mail: team@spluseins.de <b>Haftungsausschluss: Haftung für Inhalte</b></p>
```

Comment:
```suggestion
          <p>E-Mail: team (at) spluseins (Punkt) de <b>Haftungsausschluss: Haftung für Inhalte</b></p>
```
Hält die dümmsten Spambots fern